src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimantResponseService.java

```
@@ -74,8 +80,25 @@ public void save(
         claimantResponseRule.assertCanBeRequested(claim, claimantId);
 
         Claim updatedClaim = caseRepository.saveClaimantResponse(claim, claimantResponse, authorization);
+
         claimantResponseRule.isValid(updatedClaim);
         formaliseResponseAcceptance(claimantResponse, updatedClaim, authorization);
+
+        Response response = claim.getResponse().orElseThrow(IllegalStateException::new);
+
+        if(isFullDefenseDisputeAcceptation(response, claimantResponse)) {
+            try {
+                appInsights.trackEvent(AppInsightsEvent.CLAIM_STAYED, REFERENCE_NUMBER, updatedClaim.getReferenceNumber());
+                caseRepository.saveCaseEvent(
+                    authorization,
+                    updatedClaim,
+                    CaseEvent.STAY_CLAIM
+                );
+            } catch (Exception e) {
+                logger.error(String.format("Error whilst staying claim %s", claim.getId()), e);
```

Rethrow this exception rather than swallowing it.